features/element-hiding.json

```diff
@@ -693,6 +699,15 @@
                     }
                 ]
             },
+            {
+                "domain": "mail.com",
```

Comment:
It looks a bit odd with the "Sponsored News" section that is just empty, can we fold that up too?